### PR TITLE
Display points of abstraction and link in view licence page summary tab

### PR DESCRIPTION
**Pull request by robertparkinson**
https://eaflood.atlassian.net/browse/WATER-4330

Migrating the licence summary page from the water-abstraction-ui to the water-abstraction-system.

Show the points of abstraction

### CODE REVIEW

**Review by Demwunz — APPROVED** [review]
👍🏽

File: app/presenters/licences/view-licence.presenter.js [hunk 1]
```diff
@@ -156,6 +147,75 @@ function _generateWarningMessage (ends) {
...
+  if (permitLicence.purposes.length > 0) {
+    permitLicence.purposes.forEach((purpose) => {
+      purpose.purposePoints.forEach((point) => {
+        const pointDetail = point.point_detail
+        if (pointDetail) {
+          abstractionPoints.push(_generateAbstractionContent(pointDetail))
+        }
+      })
+    })
+  }
```
Comment (Cruikshanks) [inline app/presenters/licences/view-licence.presenter.js:176]: Just a NIT. Do we need the check? If `permitLicences.purposes` is an empty array I think the `.forEach()` will simply never kick in with the same result. 🤷 

```suggestion
  permitLicence.purposes.forEach((purpose) => {
    purpose.purposePoints.forEach((point) => {
      const pointDetail = point.point_detail
      if (pointDetail) {
        abstractionPoints.push(_generateAbstractionContent(pointDetail))
      }
    })
  })
```

**Comment by Cruikshanks** [comment]
Oh, and one other small request. Remember to add a link to the ticket in the PR description and squashed commit as per [Pull requests](https://github.com/DEFRA/software-development-standards/blob/master/processes/pull_requests.md#start-with-a-commit).

Check out any of the previous commits for reference if you are unsure.

File: app/presenters/licences/view-licence.presenter.js (hunk 1 above)
Comment (robertparkinson) [reply app/presenters/licences/view-licence.presenter.js:176]: I mostly added it in to accommodate the unit tests 😅  but I can update them to

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**